src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java

```diff
@@ -159,6 +159,10 @@ private static void createAndCheckWritePermissionsFor(FileSystem fileSystem, Str
      * Returns a map with vendor ID as a key and a set of purposes as a value for given vendor list version.
      */
     public Future<Map<Integer, V>> forVersion(int version) {
+        if (version == -1) {
```

Comment:
We can do version checking as `<= 0` (see upper comment).
Also, i'd like see more granular message here, with included input version, like: `Vendor list of {0} version is not valid`